tests/unit/backend/layers/business/test_business.py

```diff
@@ -377,13 +388,14 @@ def test_update_collection_same_doi(self):
         A collection updated with the same DOI should not trigger a Crossref call
         """
         metadata = self.sample_collection_metadata
-        links = [Link("test doi", "DOI", "http://test.doi")]
+        links = [Link("test doi", "doi", "http://test.doi")]
```

Comment:
The link type needs to move to a constant (enum) to avoid these errors. It's on my TODO list.